client/src/util/quantile.js

```diff
@@ -0,0 +1,26 @@
+/*
+quantiles - calculate quantiles for the typed array.
+
+Currently interpolates to 'higher' value.
```

Comment:
Looking at line 24, I believe it actually interpolates to the lower value. Either is valid and fine by me, just a small inconsistency.

Reply:
will fix comment.